packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1047,28 +1047,36 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
     if (_enabled) {
       items = widget.selectedItemBuilder == null
         ? List<Widget>.from(widget.items)
-        : widget.selectedItemBuilder(context).map<Widget>((Widget item) {
-            return Container(
-              constraints: const BoxConstraints(minHeight: _kMenuItemHeight),
-              alignment: AlignmentDirectional.centerStart,
-              child: item,
-            );
-          }).toList();
+        : widget.selectedItemBuilder(context);
     } else {
-      items = <Widget>[];
+      items = widget.selectedItemBuilder == null
+        ? <Widget>[]
+        : widget.selectedItemBuilder(context);
     }
 
     int hintIndex;
     if (widget.hint != null || (!_enabled && widget.disabledHint != null)) {
-      final Widget emplacedHint = _enabled
-        ? widget.hint
```

Comment:
Previously, an app that specified a hint for an enabled DropdownButton, would not see that hint aligned and its minHeight would not be constrained (because we didn't wrap it). That seems like a pointless inconsistency, but we'll need to watch out for problems introduced by this.

Reply:
Right -- this would be the breaking change part of this PR, I think it might still be worth sending an email out to Flutter developers to notify them of the correct fix if they run into problems here